apps/api/src/server/applications/application/application.routes.js

```diff
@@ -29,14 +29,39 @@ router.post(
 	asyncHandler(createApplication)
 );
 
-router.post('/:id/start', validateApplicationId, asyncHandler(startCase));
+router.post(
+	'/:id/start',
+	/*
+        #swagger.tags = ['Applications']
+        #swagger.path = '/applications/{id}/start'
+        #swagger.description = 'Moves application from Draft state to Pre-Application state'
+        #swagger.parameters['id'] = {
+            in: 'path',
+            description: 'Application ID',
+            required: true,
+            type: 'integer'
+        }
+        #swagger.responses[200] = {
+            description: 'Application Details',
+            schema: { id: 1, reference: 'AB01102030', status: 'Pre-Application'}
```

Comment:
reference is 1 digit too long.  would recommend something like 'EN0110203' - ie case 10203 of type EN01.

Reply:
Ah yeah, I just smashed the keyboard as this is swagger documentation. Will change now